test/integration/targets/mongodb_shard/files/bash/ensure_primary.sh.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This will fix the CI issue with the shebang:

```suggestion
#!/usr/bin/env bash
```

Reply:
OK thanks. I thought the first one was in the allowed list. Must have misunderstood something.